Index: linux-4.12-rc4/include/linux/skbuff.h
===================================================================
--- linux-4.12-rc4.orig/include/linux/skbuff.h
+++ linux-4.12-rc4/include/linux/skbuff.h
@@ -753,6 +753,7 @@ struct sk_buff {
 	__u8			ipvs_property:1;
 	__u8			inner_protocol_type:1;
 	__u8			remcsum_offload:1;
+	__u8			fast_forwarded:1;
 #ifdef CONFIG_NET_SWITCHDEV
 	__u8			offload_fwd_mark:1;
 #endif
