Index: linux-4.12-rc4/net/netfilter/nf_conntrack_proto_tcp.c
===================================================================
--- linux-4.12-rc4.orig/net/netfilter/nf_conntrack_proto_tcp.c
+++ linux-4.12-rc4/net/netfilter/nf_conntrack_proto_tcp.c
@@ -33,6 +33,10 @@
 #include <net/netfilter/ipv4/nf_conntrack_ipv4.h>
 #include <net/netfilter/ipv6/nf_conntrack_ipv6.h>
 
+/* Do not check the TCP window for incoming packets  */
+static int nf_ct_tcp_no_window_check __read_mostly = 1;
+EXPORT_SYMBOL_GPL(nf_ct_tcp_no_window_check);
+
 /* "Be conservative in what you do,
     be liberal in what you accept from others."
     If it's non-zero, we mark only out of window RST segments as INVALID. */
